### PR TITLE
Prepare release 1.2.0

### PR DESCRIPTION
**Pull request by hartwork**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -461,7 +461,7 @@ uses Graphviz too:
 Changelog
 ---------
 
-* ``UPCOMING`` — XXXX-XX-XX
+* ``v1.2.0`` — 2022-02-28
```
Comment (esc) [inline README.rst:464]: It's now march.

File: git-big-picture.1 [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man.
-.TH GIT-BIG-PICTURE "1" "January 2021" "git-big-picture 1.1.1" "User Commands"
+.TH GIT-BIG-PICTURE "1" "February 2022" "git-big-picture 1.2.0" "User Commands"
```
Comment (esc) [inline git-big-picture.1:2]: also march

**Review by esc — CHANGES_REQUESTED** [review]
I trawled the source code for any other references to the version. Seems good to me. Only the month needs updating and this will be ready for launch! 🚀

**Comment by hartwork** [comment]
@esc merged and released. If you like to tweet about it, please feel encouraged to

**Comment by esc** [comment]
> @esc merged and released. If you like to tweet about it, please feel encouraged to

Sure: https://twitter.com/esc___/status/1499060753285926921

**Pull request merged**